src/spyglass/position/v1/dlc_utils.py

```diff
@@ -19,10 +19,14 @@
 from position_tools import get_distance
 
 from spyglass.common.common_behav import VideoFile
+from spyglass.common.common_position import PositionVideo
 from spyglass.common.common_usage import ActivityLog
 from spyglass.settings import dlc_output_dir, dlc_video_dir, raw_dir
 from spyglass.utils.logging import logger, stream_handler
 
+convert_to_pixels = PositionVideo.convert_to_pixels
```

Comment:
If consoldiating functions anyways, I think this is an odd table to depend on (no entries in the database).  Would these be better in a position_utils or just independent functions instead of methods?

Reply:
Good point!